### ISSUE TITLE
chore: cut v0.16 release

### ISSUE THREAD

**Issue by lizthegrey** [open-telemetry/opentelemetry-js-contrib#448]
### Is your feature request related to a problem? Please describe
v0.19 of opentelemetry-js is out, and it would be nice to be able to upgrade contrib at the same time to sync on using the same api/sdk deps/peerdeps.

it might be even better to sync the version numbering to get contrib and main back in sync, as well e.g. skipping v0.16 through v0.18 and going straight to v0.19

**Comment by vmarchaud** [comment]
I've tried to release it but we have a bug in our automation sadly, we'll need to wait for other maintainers to step in

> it might be even better to sync the version numbering to get contrib and main back in sync, as well e.g. skipping v0.16 through v0.18 and going straight to v0.19

We tried to do that in the past but it was quite hard to follow (since we don't necessarily have the cadence between both) but i'm pretty sure we could automate this at some point
